src/components/td/td-styles.jsx

```diff
@@ -20,12 +20,61 @@ export default theme => {
       minWidth: 20,
       paddingTop: 8,
       paddingBottom: 8,
+      display: 'block',
+
+      '&::before': {
+        display: 'block',
+        content: 'attr(data-title)',
+      },
 
       [mixins.media('md')]: {
         fontSize: 14,
+        display: 'inline-block',
         minWidth: 40,
         paddingTop: 10,
         paddingBottom: 10,
+        '&::before': {
+          content: 'none',
+        },
+      },
+
+      '&.flexOrder': {
+        order: props => props.flexOrder,
+
+        [mixins.media('md')]: {
+          order: () => 1,
+        },
+      },
+
+      '&.flexBasis': {
+        flexBasis: props =>
+          props.flexBasisMobile
+            ? `${props.flexBasisMobile}${typeof props.flexBasisMobile === 'number' ? '%' : ''}`
+            : `${props.flexBasisDesktop}${typeof props.flexBasisDesktop === 'number' ? '%' : ''}`,
+
+        [mixins.media('md')]: {
+          flexBasis: props => `${props.flexBasisDesktop}${typeof props.flexBasisDesktop === 'number' ? '%' : ''}`,
+        },
+      },
+
+      '&.width': {
+        width: props => `${props.width}${typeof props.width === 'number' ? '%' : ''}`,
+      },
+
+      '&.align': {
+        textAlign: props => (props.alignMobile ? props.alignMobile : props.alignDesktop || props.align),
+
+        [mixins.media('md')]: {
+          textAlign: props => props.alignDesktop || props.align,
+        },
+      },
+
+      '&.hidden': {
+        display: props => (props.hiddenOnMobile ? 'none' : 'block'),
+
+        [mixins.media('md')]: {
+          display: props => (props.hiddenOnDesktop ? 'none' : 'inline-block'),
+        },
```

Comment:
I think this could be:
```
hiddenOnDesktop: {
  [mixins.media('md')]: {
    display: 'none',
  },
},
hiddenOnMobile: {
  [mixins.media('sm')]: {
    display: 'none',
  },
},
```

Reply:
No, If I use :
  [mixins.media('sm')]: {
    display: 'none',

it will hide for all SM and above (MD, LG) so  including PC screens

Reply:
Right, we need a mixin to target mobile specifically if we were to do this implementation. Such a solution seems a better path. In robosave we use
```
onMobileOnly: `@media (max-width: ${theme.breakpoints.sm - 1}px)`,
onDesktop: `@media (min-width: ${theme.breakpoints.sm}px)`,
```
@bstream what would you think of doing the same in `ui-kit`?

Reply:
Captured as part of #346